src/Test.UnitTests.BinaryParsers/PEBinary/PEBinaryTests.cs

```diff
@@ -32,27 +32,40 @@ internal static string GetTestDirectory(string relativeDirectory)
             return Path.Combine(dirPath, relativeDirectory);
         }
 
-        [Fact]
-        public void PEBinary_PdbAvailable()
+        [Theory]
+        [InlineData("clangcl.14.pe.c.codeview.pdbpagesize_default.exe")]
```

Comment:
>14 [](http://example.com/codeflow?start=29&length=2)

The only version of test is 14 because it only starts support at v14, and the LLVM v15 is not yet in released status.